core/src/main/scala/pureconfig/error/ConfigReaderFailure.scala

```diff
@@ -69,8 +84,12 @@ final case object CannotConvertNull extends ConfigReaderFailure {
  * @param because the reason why the conversion was not possible
  * @param location an optional location of the ConfigValue that raised the
  *                 failure
+ * @param path an optional path to the value that couldn't be converted
```

Comment:
Unfortunately, I don't have a better idea. I ended up calling it path because that's the name Typesafe Config uses, but I agree it can be ambiguous.